CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Fixed
 
 - Crash due to assertion failure on 32-bit architectures
+- IME composition preview not appearing on Windows
```

Comment:
Wait, there wasn't any IME support before this, right?

So this should probably be `- IME support on Windows` in the `Added` section.